.github/CONTRIBUTING.md

```diff
@@ -178,6 +178,58 @@ Releases are managed by GitHub Actions, here’s an overview of the process to r
 
 If you don’t have access to the transloadit.com source code ping @arturi or @goto-bus-stop and we’ll pick it up. :sparkles:
 
+### Releasing Companion hotfix patch
```

Comment:
Shouldn't this apply to any package that we might want to hotfix previous major of?

Reply:
yes but then we have to write docs (and test) for other packages too, and probably fix bugs like `publishOnly` -> `prepack`... I didn't want to spend time on that now

Reply:
Ok, I'll add some info about other packages too